Attendance/Projects/Presentation/Presentation/Sources/CoreMember/QrCode/View/QrCodeView.swift

```diff
@@ -13,151 +13,145 @@ import CoreImage.CIFilterBuiltins
-                }
-            } else {
-                RoundedRectangle(cornerRadius: 8)
-                    .fill(Color.gray800.opacity(0.4))
-                    .frame(width:  200, height: 200)
-                    .overlay {
-                        VStack {
-                            Spacer()
-                            Image(asset: .qrCode)
-                                .resizable()
-                                .scaledToFit()
-                                .frame(width: 96, height: 96)
-                            Spacer()
-                        }
-                        
-                    }
-            }
+  
+  @ViewBuilder
+  fileprivate func generateQrImage() -> some View {
+    VStack {
+      
+      Spacer()
+        .frame(height: UIScreen.screenHeight * 0.2)
+      
+      if ((store.eventID?.isEmpty) != nil) {
+        if let qrCodeImage = store.qrCodeImage {
+          qrCodeImage
+            .interpolation(.none)
+            .resizable()
+            .scaledToFit()
+            .clipShape(RoundedRectangle(cornerRadius: 8))
+            .frame(width: 200, height: 200)
+        } else {
+          Image(asset: .appLogo)
+            .resizable()
+            .scaledToFit()
+            .frame(width: 200, height: 200)
         }
-    }
-    
-    @ViewBuilder
-    fileprivate func qrCodeReaderText() -> some View {
-        if store.eventModel != [ ] {
+      } else {
+        RoundedRectangle(cornerRadius: 8)
+          .fill(Color.gray800.opacity(0.4))
+          .frame(width:  200, height: 200)
+          .overlay {
             VStack {
-                Spacer()
-                    .frame(height: UIScreen.screenHeight * 0.1)
-                
-                Text(store.qrCodeReaderText)
-                    .pretendardFont(family: .Bold, size: 20)
-                
-                Spacer()
+              Spacer()
+              Image(asset: .qrCode)
+                .resizable()
+                .scaledToFit()
+                .frame(width: 96, height: 96)
+              Spacer()
             }
-        }
-    }
-    
-    @ViewBuilder
-    fileprivate func creatEventButton() -> some View {
-        if store.eventID?.isEmpty == nil  {
             
-            VStack {
-                Spacer()
-                    .frame(height: UIScreen.screenHeight * 0.25)
-                
-                Text("일정 등록하기")
-                    .pretendardFont(family: .SemiBold, size: 16)
-                    .foregroundColor(Color.gray300)
-                    .underline(true, color: Color.gray300)
-                    .onTapGesture {
-                        store.send(.navigation(.presentSchedule))
-                    }
-                
-                
-                Spacer()
-                    .frame(height: 10)
-            }
-        }
+          }
+      }
+    }
+  }
+  
+  @ViewBuilder
+  fileprivate func qrCodeReaderText() -> some View {
+    if store.eventModel != [ ] {
```

Comment:
```suggestion
    if store.eventModel.isEmpty == false {
```
빈 배열이랑 비교하기 보다 이렇게 수정하는게 가독성이 더 좋을 것 같아요!
`ScheduleEventView` 에서도 동일한 코드가 있던데 같이 수정 부탁드립니다!